src/main/resources/application.yaml

```diff
@@ -16,3 +16,15 @@ management:
 spring:
   application:
     name: Bulk Scan Payment Processor
+
+site-configuration:
+  sites:
+    - siteName: PROBATE
+      poBox: 12625
+      siteId: AA08
+    - siteName: DIVORCE
+      poBox: 12706
+      siteId: AA07
+    - siteName: DIVORCE
+      poBox: 12746
+      siteId: AA07
```

Comment:
I don't think `siteId` should be exposed 🙊

Reply:
why do you think so ?

Reply:
This is basically site identifier. I don't think such information should ever go public

Reply:
Got confirmation from Fees and Pay and Salman that site id should not be public. Please store them in key vault. Also now you will need to remove git commit history as it is already public in PR